back/Dockerfile

```diff
@@ -31,14 +31,14 @@ RUN apt-get update                                        \
    # Required by `pg` gem
    libpq-dev                                              \
    # Used to wait for Postgres to be up in CircleCI
-   netcat                                                 \
+   netcat-openbsd                                         \
```

Comment:
Why openbsd? Doesn't this image use Debian?

Reply:
Well, I must admit I didn't dig that deep into it. `apt-get update` was failing in the new image because, reportedly, netcat was a virtual package and there was no installation candidate for it. Stackoverflow told me we basically had the choice between netcat-traditional and netcat-openbsd. After a quick kagi check, netcat-traditional seems to be more of a legacy thing. `netcat-openbsd` is a rewrite by openbsd, but I don't think it's specific to OpenBSD.